docs/content/en/docs/migrate/evolution-klt/_index.md

```diff
@@ -0,0 +1,192 @@
+from build to test to deployment,
+including notification tools and monitoring tools,
+All these tools are connected
+through the CloudEvents open standard.
+
+This enables more automated delivery and remediation
+based on observability data.
+Observability is in the center of Keptn.
+Keptn v1 connects different testing and evaluation tools
+to execute sequences
+that ensure that deployments get into production safely.
+
+The quality gates feature enables you to define
+sophisticated analyses of the health of your deployment
+at each stage.
+They use Service-Level Indicators (SLIs),
+which is information provided as queries to your monitoring software
+(such as Prometheus, Dynatrace, or DataDog),
+and allow you to define Service Level Objectives (SLOs)
+which can be based on sophisticated calculations
+of multiple SLIs over multiple build/test cycles.
+SLIs can be weighted and the results presented
+as a general score of the health of your deployment.
+
+Keptn v1 also supports a remediation use case
+that can react to a problem such as a Prometheus alert,
+and execute sequences of tasks
+that bring the system back to a healthy state.
+Remediation calls your tools and looks at observability data
+from whatever data source you choose --
+Prometheus, Dynatrace, DataDog --
+to see if the system got back to a healthy state.
+
+## Keptn Lifecycle Toolkit (KLT)
+
+Development of the Keptn Lifecycle Toolkit began in 2022
+in response to industry trends and feedback from Keptn users.
+
+Specifically, Cloud Native GitOps has shifted complexity
+from devopment to operations.
+GitOps promotes a microservice-based paradigm
+where each development team runs,configures,
+and builds their own Git repository
+then uses tools such as Argo and Flux to push
+their microservice out to the target environment.
+This makes it easier for developers to work on their individual services:
+they become more innovative and deploy faster.
+but more complex for people who must operate these apps.
+
+Operations is now responsible for determining
+exactly what the application is --
+is it these five services in a certain version?
+Is it another flavor of the application
+with a different version of one service?
+Who is resposible for the individual tests
+as you start to deploy these different services together?
+
+GitOps deploys a service into the environment
+but testing, security checks, and validation are not part of GitOps.
+Different teams using different tools
+and different flavors of Kubernetes
+make it more complex for operations
+to validate the health of an application,
+ensuring that everything is secure, everything is orchestratable,
+and everything is observable.
+
+Keptn v1 can define metrics, response time, failure rates,
+and use a variety of tools
+to validate whether your overall deployment is actually healthy.
+But it has flaws in the GitOps world:
+
+- Single-pod ready does not mean application ready.
```

Comment:
This is not a flaw of Keptn LTS. It's a truism about pods - even if Keptn wasn't involved at all.

Reply:
Added note to source file